backend/prisma/schema.prisma

```diff
@@ -69,7 +70,9 @@ model Plan {
   specialEventType EventType
   createdAt        DateTime         @default(now())
   updatedAt        DateTime         @updatedAt
-  PlanMembership   PlanMembership[]
+  planMembership   PlanMembership[]
+  contacts         Contact[]
```

Comment:
If we're trying to establish that one plan can only be associated to one contact, this `contacts    Contact[]` should be removed. Instead we should have:

```js
model Plan {
   other fields...
   contact          Contact          @relation(fields: [contactID], references: [id])
   contactID        String
}

Reply:
this makes sense done!